jobs/PackerCacheBuild/Jenkinsfile

```diff
@@ -0,0 +1,68 @@
+import groovy.transform.Field;
```

Comment:
[!!!!!]
I recalled another more important thing, there're 2 triggers  of the "cache-build" job:
(1) any SCM change in RackHD/packer folder
```
checkout(
[
  $class: 'GitSCM',
  branches: [[name: '*/master']],
  doGenerateSubmoduleConfigurations: false,
  extensions: [[$class: 'PathRestriction', excludedRegions: '', includedRegions: 'packer/(.*/)?.*']],
  submoduleCfg: [], userRemoteConfigs: [[url: 'https://github.com/RackHD/RackHD/']]
]
)
```
(2) We need to specific the "Build Trigger" to "daily" in the Jenkins job configuration as well.